src/KafkaFlow.Retry/Durable/Definitions/Builders/Polling/PollingDefinitionsAggregatorBuilder.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace KafkaFlow.Retry
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using KafkaFlow.Retry.Durable.Definitions.Polling;
+
+    public class PollingDefinitionsAggregatorBuilder
+    {
+        private readonly List<PollingDefinition> pollingDefinitions;
+        private string schedulerId;
+
+        public PollingDefinitionsAggregatorBuilder()
+        {
+            this.pollingDefinitions = new List<PollingDefinition>();
+        }
+
+        public PollingDefinitionsAggregatorBuilder WithCleanupPollingConfiguration(Action<CleanupPollingDefinitionBuilder> configure)
+        {
+            var cleanupPollingDefinitionBuilder = new CleanupPollingDefinitionBuilder();
+            configure(cleanupPollingDefinitionBuilder);
+            var cleanupPollingDefinition = cleanupPollingDefinitionBuilder.Build();
+
+            this.pollingDefinitions.Add(cleanupPollingDefinition);
+
+            return this;
+        }
+
+        public PollingDefinitionsAggregatorBuilder WithRetryDurablePollingConfiguration(Action<RetryDurablePollingDefinitionBuilder> configure)
+        {
+            var retryDurablePollingDefinitionBuilder = new RetryDurablePollingDefinitionBuilder();
+            configure(retryDurablePollingDefinitionBuilder);
+            var retryDurablepollingDefinition = retryDurablePollingDefinitionBuilder.Build();
+
+            this.pollingDefinitions.Add(retryDurablepollingDefinition);
+
+            return this;
+        }
+
+        public PollingDefinitionsAggregatorBuilder WithSchedulerId(string schedulerId)
```

Comment:
Is null or empty `schedulerId` a valid value?

Reply:
We are validating the `schedulerId` in the `PollingDefinitionsAggregator` constructor. Should we also add it here?